packages/cli/src/commands/seed.ts

```diff
@@ -0,0 +1,59 @@
+import {Command} from '@oclif/command'
+import {join} from 'path'
+import pkgDir from 'pkg-dir'
+import {log} from '@blitzjs/display'
+import {runMigrate} from './db'
+
+const projectRoot = pkgDir.sync() || process.cwd()
+
+const seedFunctions = ['default', 'seed']
+
+export class Seed extends Command {
```

Comment:
would you be able to write some tests for this class? my goal is to have unit tests for as much of our new CLI code as we can

Reply:
@aem take a look now, seems quite verboseas I also added the full migrations tests. I'm removing it now and just mocking the `runMigration` function

Reply:
Actually I'm getting a lot of weird errors tied to Oclif, so I'll leave it like this